tests/unit/test_dict_merger.py

```diff
@@ -244,3 +341,95 @@ def test_data_lists_bare_lists():
     m.merge()
     assert m.merged_root == u
     assert not m.skipped_lists
+
+
+def test_get_custom_strategies():
```

Comment:
Since this test is working on deep logic, some comments would really be helpful to maintain it (as well as the following tests).